onnxruntime/core/providers/cpu/tensor/unsqueeze.cc

```diff
@@ -31,12 +41,26 @@ Status UnsqueezeBase::PrepareCompute(OpKernelContext* ctx, Prepare& p) const {
   ORT_ENFORCE(X != nullptr);
   auto& input_tensor = *X;
 
-  // New dimension count is the current dimensions + the number of entries in axes_
+  std::vector<int64_t> axes;
```

Comment:
Will wait for https://github.com/microsoft/onnxruntime/pull/5691 to go through and re-base with master after that change